src/main/scala/com/microsoft/kusto/spark/datasource/KustoOptions.scala

```diff
@@ -10,6 +13,13 @@ object KustoOptions {
     name
   }
 
+  val KEY_VAULT_URI = "keyVaultUri"
+  val KEY_VAULT_CREDENTIALS = "keyVaultCredentials"
+  val KEY_VAULT_PEM_FILE_PATH = "keyVaultPemFilePath"
```

Comment:
KEY_VAULT_CERTIFICATE_PATH (same for password).
We can add in the comments that the certificate is expected to be in PEM format